waku/v2/node/waku_node.nim

```diff
@@ -391,11 +390,11 @@ proc registerRelayDefaultHandler(node: WakuNode, topic: PubsubTopic) =
   if node.wakuRelay.isSubscribed(topic):
     return
 
-  proc traceHandler(topic: PubsubTopic, data: seq[byte]) {.async, gcsafe.} =
+  proc traceHandler(topic: PubsubTopic, msg: WakuMessage) {.async, gcsafe.} =
     trace "waku.relay received",
       peerId=node.peerId,
       pubsubTopic=topic,
-      hash=MultiHash.digest("sha2-256", data).expect("valid hash").data.buffer.to0xHex(), # TODO: this could be replaced by a message UID
+      hash="0x" & $topic.digest(msg).toHex(),
```

Comment:
A minor request. There is a [`to0xHex()` proc](https://github.com/status-im/nim-stew/blob/e18f5a62af2ade7a1fd1d39635d4e04d944def08/stew/byteutils.nim#L232) in `stew/byteutils`. Could you replace the string concat here with that method?

Reply:
sure! fair request. done
e41d04253d2a4c03eb72c73a603ae53f69c9af3f